packages/now-ruby/test/fixtures/09-activesupport/Gemfile

```
@@ -0,0 +1,3 @@
+source "https://rubygems.org"
+
+gem 'activesupport', '~> 6.0'
```

```suggestion
gem "activesupport", "~> 6.0"
```

Our other test fixtures use double quotes here.